packages/react-components/react-button/src/components/Button/useButtonStyles.ts

```diff
@@ -19,8 +19,10 @@ const useRootStyles = makeStyles({
   // Base styles
   base: {
     alignItems: 'center',
+    boxSizing: 'border-box',
```

Comment:
Is this needed? I think `display: inline-flex` already makes the box-sizing `border-box`.

Reply:
Do you have a source for this? I don't think `inline-flex` applies `border-box`: https://codepen.io/seanms/pen/gOzEmgE

Reply:
Yeah, this is needed for `anchor` tags, otherwise, the button grows by a ton